test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -58,6 +58,10 @@ public ServerContext build() {
                 .put("--single", single)
                 .put("--memory", memory)
                 .put("--Threads", numThreads)
+                .put("--base-server-threads", numThreads)
+                .put("--layout-server-threads", numThreads)
```

Comment:
Is this necessary if the LayoutServer is single-threaded?

Reply:
Correct. Removed it.